packages/@aws-cdk/aws-rds/lib/cluster.ts

```diff
@@ -484,6 +482,9 @@ export class DatabaseCluster extends DatabaseClusterNew {
   private readonly singleUserRotationApplication: secretsmanager.SecretRotationApplication;
   private readonly multiUserRotationApplication: secretsmanager.SecretRotationApplication;
 
+  public readonly instanceIdentifiers: string[] = [];
+  public readonly instanceEndpoints: Endpoint[] = [];
```

Comment:
Let's not have these have default values. They're not used anyway, and without them the compiler will make sure these get initialized. So:

```suggestion
  public readonly instanceIdentifiers: string[];
  public readonly instanceEndpoints: Endpoint[];
```